ocaml-lsp-server/bench/ocaml_lsp_bench.ml

```diff
@@ -0,0 +1,44 @@
+open Core
+open Core_bench
+open Ocaml_lsp_server.Testing
```

Comment:
Don't open this module and just qualify everything you need as `Testing.` please.